app/src/renderer/components/common/TmSessionSignUp.vue

```diff
@@ -233,7 +249,7 @@ export default {
       signUpName: { required, minLength: minLength(5) },
       signUpPassword: { required, minLength: minLength(10) },
       signUpPasswordConfirm: { sameAsPassword: sameAs(`signUpPassword`) },
-      signUpWarning: { required },
+      signUpWarning: { sameAs: sameAs(() => true) },
```

Comment:
Same as TmSessionAccountDelete, here we are checking the fields.signupWarning.required at line 120

Reply:
nice catch